### DISCUSSION TITLE
Inline display formatted strings

### DISCUSSION THREAD

**Discussion by nandoconde** [fonsp/Pluto.jl#1820] [Questions about using Pluto] [answered]
Hello!

Is there any way to display formatted strings (with `@sprintf`) inside a Markdown block without using with_terminal? Ideally, something like this:

```julia
md"This is my variable with limited decimals: $(Print(@sprintf "%5.4f" BigFloat("6.666666666666666")))"
```

**Comment by ctrekker** [comment, marked as answer]
Hi nandoconde!

I would suggest two changes:
1. Since `@sprintf` is resolved into a string already, the `Print()` call around it can be removed
2. Replace the single quotes of the markdown string with triple quotes

These two changes ended up producing the desired result for me.

```julia
md"""This is my variable with limited decimals: $(@sprintf "%5.4f" BigFloat("6.666666666666666"))"""
```

**Reply by nandoconde** [reply]
Thanks!! This worked perfectly
